packages/prisma/migrations/20220323033335_reschedule_fields_to_bookings_table/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- AlterTable
+ALTER TABLE "Booking" ADD COLUMN     "fromReschedule" TEXT,
+ADD COLUMN     "rescheduled" BOOLEAN;
```

Comment:
Is there benefit for using `rescheduledAt` instead? since IMO it can be rescheduled multiple times.

Reply:
Right now as I told you before original booking gets deleted on reschedule so I would say no at the moment. :(